src/server/pfs/cmds/cmds.go

```diff
@@ -771,6 +771,27 @@ The squash will fail if it includes a commit with no children`,
 	shell.RegisterCompletionFunc(squashCommit, shell.BranchCompletion)
 	commands = append(commands, cmdutil.CreateAlias(squashCommit, "squash commit"))
 
+	deleteCommit := &cobra.Command{
+		Use:   "{{alias}} <commit-id>",
+		Short: "Delete the sub-commits of a commit.",
```

Comment:
feels a little weird that only squash and delete use "sub-commit" now but I don't really have a suggestion